pkg/kernel/Dockerfile

```diff
@@ -9,6 +9,9 @@ ARG BUILD_PKGS_BASE="argp-standalone automake bash bc binutils-dev bison build-b
 # we use the same image in several places
 ARG EVE_ALPINE_IMAGE=lfedge/eve-alpine:9fb9b9cbf7d90066a70e4704d04a6fe248ff52bb
 
+# PREEMPT_RT support
+ARG PREEMPT_RT_SUPPORT=false
```

Comment:
This is a string, right? So shouldn't the default be an empty string?

Reply:
We need the default value to satisfy sbom, which fails if nothing is defined, because there is a FROM line:
```
FROM kernel-build-preempt-rt-${PREEMPT_RT_SUPPORT} AS kernel-build
```

Reply:
OK